packages/@functionless/asl-graph/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@functionless/asl-graph",
+  "version": "0.27.0",
```

Comment:
shouldn't version be 0? do we have a doc on how these work?

Reply:
If i recall, it doesn't matter what we put the first version at. Maybe it should be 0.0.0, can't remember. @tvanhens ?